FoundationBuilder/include/Application.h

```diff
@@ -150,4 +150,13 @@ class Application
 
 private:
     Application() {}
+
+    std::string inputFilepath; //path to input .STL model
```

Comment:
Think about making CommandLine object the member of Application class. Thus, you won't need duplicate of all these parameters.

Reply:
i have created `CommandLine cl` field in `Application class`. And now i must announce all parameters inside `Application::run()`? if i announce them inside `Application::init()`, they will be wisible only inside `run()` method.

![image](https://user-images.githubusercontent.com/45259941/84596347-cd72bc80-ae65-11ea-8c11-9a3e705806ed.png)

Reply:
I mean, you don't need these local `std::string inputfilepath`, `float height`, etc. duplicates, since you can just use `cl.getValueAs()` in all places, where you need parameter values

Reply:
Ok, I misunderstood firstly